package.json

```diff
@@ -1,7 +1,38 @@
 {
+  "license": "MIT",
   "dependencies": {
     "form-data": "^2.1.2",
     "node-fetch": "^1.6.3",
-    "syncano-server": "^0.7.2-43"
+    "syncano-server": "beta"
+  },
+  "scripts": {
+    "test": "yarn && npm-run-all test:lint test:e2e:coverage",
```

Comment:
Remove `npm-run-all`, just replace it with `npm run test:lint && npm run test:e2e:coverage`. We don't need a package only for this.